### PR TITLE
feat: logs list view changes

### PR DESCRIPTION
**Pull request by Vikrant2520**
### Summary

- Change Auto Refresh dropdown according to new designs
- Handle the list view changes for raw / default and table view
- remove the popover arrow icons from the date time popover
- remove the shadows from the antd buttons for toolbar
- raw view changes for the logs listing
- default view changes for the logs listing

#### Screenshots

Screenshot 2024-01-09 at 5 53 36 PM

https://github.com/SigNoz/signoz/assets/54737045/39f39e3f-1d1e-4b5a-ac03-33edf5b72870

### CODE REVIEW

File: frontend/src/components/Logs/ListLogView/ListLogView.styles.scss [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.log-selected-fields {
+	display: flex;
+	width: 100%;
+	overflow: hidden;
+	align-items: center;
+
+	.selected-log-field-key {
+		color: #7190f9 !important;
```
Comment (YounixM) [inline frontend/src/components/Logs/ListLogView/ListLogView.styles.scss:37]: Use from ui library

File: frontend/src/components/Logs/ListLogView/index.tsx [hunk 2]
```diff
@@ -145,82 +146,59 @@ function ListLogView({
...
+	};
+
 	return (
-		<Container $isActiveLog={isHighlighted}>
-			<div>
-				<LogContainer>
-					<>
-						<LogGeneralField fieldKey="log" fieldValue={flattenLogData.body} />
+		<Container
+			$isActiveLog={isHighlighted}
```
Comment (YounixM) [inline frontend/src/components/Logs/ListLogView/index.tsx:161]: $isActiveLog ?

File: frontend/src/components/Logs/LogLinesActionButtons/LogLinesActionButtons.styles.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	.ant-btn-default {
+		border: none;
+		box-shadow: none;
+		padding: 9px;
+		justify-content: center;
+		align-items: center;
+		display: flex;
+
+		&.active-tab {
+			background-color: #1d212d;
```
Comment (YounixM) [inline frontend/src/components/Logs/LogLinesActionButtons/LogLinesActionButtons.styles.scss:21]: Use from ui library

File: frontend/src/container/TopNav/AutoRefreshV2/AutoRefreshV2.styles.scss [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	}
+}
+.auto-refresh-menu {
+	display: flex;
+	flex-direction: column;
+}
+
+.auto-refresh-checkbox {
+	padding: 12px 14px;
+	border-bottom: 1px solid #1d212d;
```
Comment (YounixM) [inline frontend/src/container/TopNav/AutoRefreshV2/AutoRefreshV2.styles.scss:27]: Use from ui library

File: frontend/src/container/TopNav/AutoRefreshV2/index.tsx [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+				);
+			}
+			setIsAutoRefreshfreshEnabled(checked);
+		},
+		[localStorageData, pathname],
+	);
+
+	if (globalTime.selectedTime === 'custom') {
+		// eslint-disable-next-line react/jsx-no-useless-fragment
+		return <></>;
```
Comment (YounixM) [inline frontend/src/container/TopNav/AutoRefreshV2/index.tsx:138]: Check if we can avoid this.

**Review by YounixM — APPROVED** [review]
Added minor comments. Rest LGTM

PR is missing lightMode styles. Please take care of this too.

File: frontend/src/components/Logs/ListLogView/ListLogView.styles.scss (hunk 1 above)
Comment (Vikrant2520) [reply frontend/src/components/Logs/ListLogView/ListLogView.styles.scss]: done

File: frontend/src/components/Logs/ListLogView/ListLogView.styles.scss [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+		font-size: 14px;
+		font-style: normal;
+		font-weight: 400;
+		line-height: 18px; /* 128.571% */
+		letter-spacing: -0.07px;
+	}
+
+	.selected-log-value {
+		color: #ad7f58;
```
[the remark this answers is not in the document]
Comment (Vikrant2520) [reply frontend/src/components/Logs/ListLogView/ListLogView.styles.scss]: done

File: frontend/src/components/Logs/ListLogView/index.tsx (hunk 2 above)
Comment (Vikrant2520) [reply frontend/src/components/Logs/ListLogView/index.tsx:161]: this is for styled components styling , existing code

File: frontend/src/components/Logs/LogLinesActionButtons/LogLinesActionButtons.styles.scss (hunk 3 above)
Comment (Vikrant2520) [reply frontend/src/components/Logs/LogLinesActionButtons/LogLinesActionButtons.styles.scss]: done

File: frontend/src/container/TopNav/AutoRefreshV2/AutoRefreshV2.styles.scss (hunk 4 above)
Comment (Vikrant2520) [reply frontend/src/container/TopNav/AutoRefreshV2/AutoRefreshV2.styles.scss]: done

File: frontend/src/container/TopNav/AutoRefreshV2/index.tsx (hunk 5 above)
Comment (Vikrant2520) [reply frontend/src/container/TopNav/AutoRefreshV2/index.tsx:138]: this is part of existing code , will refactor a bit in next PR will check if we can remove this

**Pull request merged**